content/en/_index.md

```diff
@@ -1,18 +1,42 @@
 ---
-title: KCD Helsinki 2025
+title: Cloud Native Finland
 toc_hide: true
 ---
 
 <div class="row mt-5 mb-3">
     <div class="col-lg-6">
         <div class="lead">
-        Kubernetes Community Days is a one day event about cloud native in Helsinki.
-        In 2025 the KCD will take place on May 6 in Valilla.
+        Cloud Native Finland is a non for profit organisation for promoting open source cloud native technologies in
+        Finland.
         </div>
     </div>
     <div class="col-lg-6">
         <img src="/images/kcd-logo-color.svg" alt="KCD logo" style="max-width: 300px;">
     </div>
 </div>
 
+# Mission
+
+Our mission is to promite cloud native technologies in Finland by sharing knowledge and create space for technologists
```

Comment:
```suggestion
Our mission is to promite cloud native technologies in Finland by sharing knowledge and creating space for technologists
```

Reply:
Fixed in [3b9a562](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/3b9a5621fba6928fcbb42463b533d9c60ad53a11)